.github/workflows/kuttl_tests.yaml

```diff
@@ -75,6 +75,16 @@ jobs:
     steps:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
+    - name: Set up Go 1.16
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.16
+    - uses: actions/cache@v2
```

Comment:
Could you give a name to this step? Otherwise it shows up as "Run actions/cache@v2".

Reply:
Done. Although I'm not sure what value the name has in this case ;)